### PR TITLE
feat(type declarations): improve type of activationStrategy

### PR DESCRIPTION
**Pull request by aluanhaddad**
Added the ActivationStrategy interface to describe the shape of the activationStrategy object that enumerates route activation strategies.

### CODE REVIEW

File: src/interfaces.js [hunk 1]
```diff
@@ -170,3 +170,21 @@ interface ConfiguresRouter {
   */
   configureRouter(config: RouterConfiguration, router: Router): Promise<void>|PromiseLike<void>|void;
 }
+
+/**
+* An optional interface describing the available activation strategies.
+*/
+interface ActivationStrategy {
+  /**
+  * Reuse the existing the view model without invoking lifecycle hooks.
```
Comment (zewa666) [inline src/interfaces.js]: The second `the` after existing should be removed in the comment. Same applies for the next method description

**Comment by zewa666** [comment]
@davismj this one should be ok for a merge

**Comment by EisenbergEffect** [comment]
@davismj Can you test whether this compiles correctly with Babel? Feel free to merge if it does.

**Comment by Vheissu** [comment]
@EisenbergEffect @davismj Can confirm this PR builds correctly under Babel.

**Pull request merged**